src/org/scada_lts/dao/PendingEventsDAO.java

```diff
@@ -142,55 +138,53 @@ public UserCommentCache mapRow(ResultSet rs, int rownumber) throws SQLException
 		return null;
 	}
 	
-	private List<EventInstance> getPendingEvents(int userId, final TreeMap<Integer, List<UserComment>> comments ) {
+	private List<EventInstance> getPendingEvents(int userId, final Map<Integer, List<UserComment>> comments ) {
 		if (LOG.isTraceEnabled()) {
 			LOG.trace("SQL PendingEvents userId:"+userId);
 		}
 		
 		try {
 			@SuppressWarnings({ "unchecked", "rawtypes" })
-			List<EventInstance> listEvents = DAO.getInstance().getJdbcTemp().query(SQL_EVENTS,new Integer[]{userId}, new RowMapper() {
-				@Override
-				public EventInstance mapRow(ResultSet rs, int rownumber) throws SQLException {
-					int typeId = rs.getInt(COLUMN_NAME_EVENT_TYPE_ID);
-					int typeRef1 = rs.getInt(COLUMN_NAME_EVENT_TYPE_REF1);
-					int typeRef2 = rs.getInt(COLUMN_NAME_EVENT_TYPE_REF2);
-					EventType type = EventTypeUtil.createEventType(typeId, typeRef1, typeRef2);
-					long activeTS = rs.getLong(COLUMN_NAME_EVENT_ACTIVE_TS);
-					Boolean rtnApplicable = DAO.charToBool(rs.getString(COLUMN_NAME_EVENT_RTN_APPLICABLE));
-					int alarmLevel = rs.getInt(COLUMN_NAME_EVENT_ALARM_LEVEL);
-					
-					LocalizableMessage message;
-					try {
-						message = LocalizableMessage.deserialize(rs.getString(COLUMN_NAME_EVENT_MESSAGE));
-					} catch (LocalizableMessageParseException e) {
-						message = new LocalizableMessage("common.default",
-								rs.getString(COLUMN_NAME_EVENT_MESSAGE));
-					}
-					
-					EventInstance event = new EventInstance(type, activeTS,	rtnApplicable, alarmLevel, message, null);
-					
-					event.setId(rs.getInt(COLUMN_NAME_EVENT_ID));
-					long rtnTs = rs.getLong(COLUMN_NAME_EVENT_RTN_TS);
-					if (!rs.wasNull())
-						event.returnToNormal(rtnTs, rs.getInt(COLUMN_NAME_EVENT_RTN_COUSE));
-					long ackTs = rs.getLong(COLUMN_NAME_EVENT_ACK_TS);
-					if (!rs.wasNull()) {
-						event.setAcknowledgedTimestamp(ackTs);
-						event.setAcknowledgedByUserId(rs.getInt(COLUMN_NAME_EVENT_ACK_USER_ID));
-						if (!rs.wasNull())
-							event.setAcknowledgedByUsername(rs.getString(COLUMN_NAME_EVENT_USERNAME));
-						event.setAlternateAckSource(rs.getInt(COLUMN_NAME_EVENT_ALTERNATE_ACK_SOURCE));
-					}
-
-					String silent = rs.getString(COLUMN_NAME_EVENT_SILENCED);
-					event.setSilenced(DAO.charToBool(silent));
+			List<EventInstance> listEvents = DAO.getInstance().getJdbcTemp().query(SQL_EVENTS,new Integer[]{userId},
+					(rs, rownumber) -> {
+				int typeId = rs.getInt(COLUMN_NAME_EVENT_TYPE_ID);
+				int typeRef1 = rs.getInt(COLUMN_NAME_EVENT_TYPE_REF1);
+				int typeRef2 = rs.getInt(COLUMN_NAME_EVENT_TYPE_REF2);
+				EventType type = EventTypeUtil.createEventType(typeId, typeRef1, typeRef2);
```

Comment:
I sugest remove variables which are used only one (lines 150 - 152) and , just  like:

EventTypeUtil.createEventType(
ts.getInt(COLUMN_NAME_EVENT_TYPE_ID,
.....

We do not have here any exception block, so any exception won't occur here. Code may be shorter and readable.